geomesa-kafka/geomesa-kafka-tools/src/main/scala/org/locationtech/geomesa/kafka/tools/KafkaRunner.scala

```diff
@@ -8,37 +8,27 @@
 
 package org.locationtech.geomesa.kafka.tools
 
-import com.beust.jcommander.JCommander
-import org.locationtech.geomesa.kafka.tools.data.{KafkaCreateSchemaCommand, KafkaRemoveSchemaCommand}
-import org.locationtech.geomesa.kafka.tools.export.{KafkaExportCommand, KafkaListenCommand}
-import org.locationtech.geomesa.kafka.tools.ingest.KafkaIngestCommand
-import org.locationtech.geomesa.kafka.tools.status._
-import org.locationtech.geomesa.tools.status._
 import org.locationtech.geomesa.tools.{Command, Runner}
 
 object KafkaRunner extends Runner {
 
   override val name: String = "geomesa-kafka"
 
-  override protected def createCommands(jc: JCommander): Seq[Command] = Seq(
-    new data.KafkaCreateSchemaCommand,
-    new data.KafkaRemoveSchemaCommand,
-    new data.KafkaUpdateSchemaCommand,
-    new export.KafkaListenCommand,
-    new export.KafkaExportCommand,
-    new ingest.KafkaIngestCommand,
-    new status.KafkaDescribeSchemaCommand,
-    new status.KafkaGetTypeNamesCommand,
-    // common commands, placeholders for script functions
-    new HelpCommand(this, jc),
-    new ConfigureCommand,
-    new ClasspathCommand,
-    new ScalaConsoleCommand,
```

Comment:
Ok, looks like the Scala console is bouncing around a bit.  Which tool bundles should it be in?

Reply:
should be in the common one